cmd/aggregated-apiserver/app/aggregated-apiserver.go

```diff
@@ -32,8 +36,21 @@ func NewAggregatedApiserverCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 
-	opts.AddFlags(cmd.Flags())
+	fss := cliflag.NamedFlagSets{}
+
+	genericFlagSet := fss.FlagSet("generic")
+	genericFlagSet.AddGoFlagSet(flag.CommandLine)
```

Comment:
I just tested it `genericFlagSet.AddGoFlagSet(flag.CommandLine)` that also not need to be added seemingly.

Reply:
@lonelyCZ , I'm curious about this, the result is the same with or without this sentence.
`// AddFlags adds flags to the specified FlagSet.
func (o *Options) AddFlags(flags *pflag.FlagSet) {
	o.RecommendedOptions.AddFlags(flags)
	flags.Lookup("kubeconfig").Usage = "Path to karmada control plane kubeconfig file."
`

Reply:
Yes, so we don't need this sentence as we don't use `flag.CommandLine`.

Reply:
Solved @lonelyCZ